app/models/runtime/user.rb

```diff
@@ -82,13 +82,13 @@ def validate
     end
 
     def validate_organization(org)
-      unless org && organizations.include?(org)
+      unless org && organizations_dataset.where(id: org).any?
```

Comment:
I might be misunderstanding something here - and this logic already existed before you touched it - but is it actually possible to have a situation where `org` is true and `organizations_dataset.where(id: org).any?` is false?